Source/Atomic/Graphics/Graphics.h

```diff
@@ -537,6 +537,9 @@ class ATOMIC_API Graphics : public Object
 
     // ATOMIC BEGIN
 
+    /// Return number of True primitives drawn this frame.
+    unsigned GetSinglePassPrimitives() const { return numSinglePassPrimitives_; }

```

Comment:
I am not sure the word "True" describes this best, it leaves me guessing at least, can this be summarized better?